src/ai_marketplace_monitor/webui/server.py

```
@@ -0,0 +1,504 @@
+        queue: asyncio.Queue[Dict[str, Any]] = asyncio.Queue(maxsize=1000)
+        log_handler.subscribe(queue)
+        try:
+            # Send a brief hello so clients know the stream is live.
+            await websocket.send_json({"type": "hello", "time": time.time()})
+            while True:
+                payload = await queue.get()
+                await websocket.send_json({"type": "log", "record": payload})
+        except WebSocketDisconnect:
+            pass
+        except Exception:
+            pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)